src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+/**
+ * Represents exceptions specific to Duke
+ *
+ * @author Aiken Wong
+ */
+public class DukeException extends RuntimeException {

```

Comment:
Personally don't think RuntimeException should be thrown here? Based on the project description, when an error is encountered, the user should be allowed to continue executing the program. You could instead inherit from the generic Exception class instead